configure.ac

```diff
@@ -302,6 +302,17 @@ AC_ARG_WITH([cc-image],
     ]
 )
 
+AC_ARG_ENABLE([autogopath],
+	AS_HELP_STRING([--enable-autogopath], [Set GOPATH variable for you, used when packaging @<:@default=false@:>@]),
```

Comment:
I'm wondering if we want to have this default to `true` (and rename and change behaviour so it can be disabled with `--disable-autogopath`). @sameo, @dlespiau - what do you think?